package.json

```diff
@@ -9,13 +9,14 @@
     "dist"
   ],
   "scripts": {
-    "dev": "vite --open",
-    "build": "tsc --p ./tsconfig-build.json && vite build",
+    "prebuild": "node ./scripts/wrapper-pre-build.cjs",
+    "dev": "npm run prebuild && vite --open",
+    "build": "npm run prebuild && tsc --p ./tsconfig-build.json && vite build",
```

Comment:
do we need `npm run prebuild` here? isn't prebuild automatically run before build by virtue of it being a "pre-script"?

Reply:
Good point, removed :)